mifospay/src/main/java/org/mifos/mobilewallet/mifospay/editprofile/ui/EditProfileActivity.java

```diff
@@ -332,7 +332,7 @@ private UCrop getConfiguredUCrop(UCrop uCrop) {
 
     @Override
     public void removeProfileImage() {
-        Toast.makeText(this,R.string.toast_profile_removed,Toast.LENGTH_SHORT).show();
+        Toast.makeText(this, R.string.toast_profile_removed, Toast.LENGTH_SHORT).show();
```

Comment:
you should use the Toaster util which is present in the project. :)

Reply:
This was to fix the gradle errors. This pr didn't mean anything to contribute